spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -11,10 +11,11 @@
 package: swiftnav.sbp.ssr
 description: Precise State Space Representation (SSR) corrections format
 stable: False
-public: True
+public: False
 include:
   - types.yaml
   - gnss.yaml
+  - observation.yaml
```

Comment:
I don't think the imports work correctly - I think they're hard-coded for the most part. Sharing structures between files is also not really a thing - it's done with gnss, but that's it. Hence the problems.